<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản Lý Thiết Bị</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #f0f0f0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Nhãn tiêu đề -->
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thông Tin Thiết Bị</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 16px; font-weight: bold;</string>
    </property>
   </widget>
   <!-- Nhãn và trường nhập liệu -->
   <widget class="QLabel" name="lblTen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập tên thiết bị</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHang">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hãng:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHang">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập hãng sản xuất</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập giá</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSoLuong">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Số lượng:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSoLuong">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập số lượng</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoTa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mô tả:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMoTa">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập mô tả</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Hình ảnh -->
   <widget class="QLabel" name="lblHinhAnh">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChonAnh">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Chọn ảnh</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #9C27B0; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Các nút chức năng -->
   <widget class="QPushButton" name="btnThem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #4CAF50; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSua">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #2196F3; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoa">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #f44336; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Tìm kiếm -->
   <widget class="QLabel" name="lblTimKiem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm kiếm:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimKiem">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tìm kiếm theo tên</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTim">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #FF9800; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Bảng thiết bị -->
   <widget class="QTableWidget" name="tableThietBi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>960</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; background-color: white;</string>
    </property>
   </widget>
   <!-- Nút thoát -->
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #607D8B; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>